cpuinfo.py

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env python
-# -*- coding: UTF-8 -*-
 
```

Comment:
Also cpuinfo.py is an external tool vendor in pytables. Please do not modify it.